build/int.cloudbuild.hpc.yaml

```
@@ -12,7 +12,167 @@
+    args:
+      ["/bin/bash", "-c", "cft test run TestFleetscope --stage init --verbose"]
+    waitFor:
+      - multitenant-apply
+  - id: fleetscope-apply
+    name: "gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS"
+    args:
+      ["/bin/bash", "-c", "cft test run TestFleetscope --stage apply --verbose"]
+    waitFor:
+      - fleetscope-init
+  - id: fleetscope-verify
+    name: "gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS"
+    args:
+      [
+        "/bin/bash",
+        "-c",
+        "cft test run TestFleetscope --stage verify --verbose",
+      ]
+    waitFor:
+      - fleetscope-apply
+
+  - id: appfactory-init
+    name: "gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS"
+    args:
+      ["/bin/bash", "-c", "cft test run TestHPCAppfactory --stage init --verbose"]
+    waitFor:
+      - fleetscope-apply
+  - id: appfactory-apply
+    name: "gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS"
+    args:
+      ["/bin/bash", "-c", "cft test run TestHPCAppfactory --stage apply --verbose"]
+    waitFor:
+      - appfactory-init
+  - id: appfactory-verify
+    name: "gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS"
+    args:
+      [
+        "/bin/bash",
+        "-c",
+        "cft test run TestHPCAppfactory --stage verify --verbose",
+      ]
+    waitFor:
+      - appfactory-apply
+
+  - id: appinfra-init
+    name: "gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS"
+    args:
+      ["/bin/bash", "-c", "cft test run TestHPCAppInfra --stage init --verbose"]
+    waitFor:
+      - appfactory-apply
+  - id: appinfra-apply
+    name: "gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS"
+    args:
+      ["/bin/bash", "-c", "cft test run TestHPCAppInfra --stage apply --verbose"]
+    waitFor:
+      - appinfra-init
+  - id: appinfra-verify
+    name: "gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS"
+    args:
+      ["/bin/bash", "-c", "cft test run TestHPCAppInfra --stage verify --verbose"]
+    waitFor:
+      - appinfra-apply
+
+  - id: monte-carlo-simulation-e2e
+    name: "gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS"
+    args:
+      [
+        "/bin/bash",
+        "-c",
+        "cft test run TestHPCMonteCarloE2E --stage verify --verbose",
+      ]
+    waitFor:
+      - appinfra-apply
+      - fleetscope-verify
+tags:
+  - "ci"
+  - "integration"
+substitutions:
+  _DOCKER_IMAGE_DEVELOPER_TOOLS: "cft/developer-tools"
+  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: "1.23"
+options:
+  machineType: E2_HIGHCPU_8
```

Could we try with `E2-STANDARD-2` (they should also provision quicker).
```suggestion
  machineType: E2-STANDARD-2
```